meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "This project provides zebkit node.js bindings."
+SUMMARY = "zebkit"
+HOMEPAGE = "https://github.com/barmalei/zebra"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
```

Comment:
@gunnarx  license  information is correct. it has 
License
Apache License, Version 2.0 http://www.apache.org/licenses/LICENSE-2.0.html

inside the repository https://github.com/barmalei/zebkit